docs/samples/advanced_config/aggregation_job_driver.yaml

```diff
@@ -87,6 +87,17 @@ worker_lease_clock_skew_allowance_secs: 60
 # (required)
 maximum_attempts_before_failure: 10
 
+# Timeout to apply when establishing connections to the helper for HTTP requests, in seconds. See
+# https://docs.rs/reqwest/latest/reqwest/struct.ClientBuilder.html#method.connect_timeout for
+# details. (optional)
```

Comment:
We should mention the default values here (i.e. omitting the option won't disable timeouts)